apps/src/aichat/redux/thunks/addChatEvent.ts

```diff
@@ -13,40 +13,46 @@ import {addEventToChatEventsCurrent} from '../slice';
 
 // This thunk adds a chat event to chatEventsCurrent (displayed in current chat workspace) if visible.
 // Then it logs the event to the backend for all chat events except notifications with includeInHistory != true.
+// It also excludes logging to both frontend and backend if the teacher is viewing as a student.
 export const addChatEvent =
   <T extends ChatEvent>(chatEvent: T) =>
   (dispatch: AppDispatch, getState: () => RootState) => {
-    // User action events are hidden from participants and only displayed in teacher view of student chat history.
+    // Do not log to frontend for UserActionEvents (eg, clear chat),
```

Comment:
nit: maybe "Do not show in front end/chat window"? Since we're technically not really logging anything with this call (just driving UI)?

Reply:
Good call, will update!